### PR TITLE
Add test suit for endpoint 8-GET/api/articles sorted by date desc

### PR DESCRIPTION
**Pull request by rcbcastillo**

### CODE REVIEW

File: app/__tests__/app.test.js [hunk 1]
```diff
@@ -227,4 +228,62 @@ describe(`GET:/api/users --sad path`, () => {
         expect(message).toBe("Invalid path");
       });
   });
-});
\ No newline at end of file
+});
+
+describe.only('GET /api/articles --happy path', () => {
```
Comment (douglashellowell) [inline app/__tests__/app.test.js]: remove `.only`s before commiting!

File: app/__tests__/app.test.js [hunk 2]
```diff
@@ -227,4 +228,62 @@ describe(`GET:/api/users --sad path`, () => {
...
+  test('200: responds with an an array of articles objects having eight properties', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({body: {articles}}) => {
+      expect(articles.length).toBeGreaterThan(0);
+      expect(Array.isArray(articles)).toBe(true);
+      expect(articles).toBeSorted({ descending: true })
```
Comment (douglashellowell) [inline app/__tests__/app.test.js]: you should specify _how_ these are sorted (should be by `created_at` by default)

File: app/__tests__/app.test.js [hunk 3]
```diff
@@ -227,4 +228,62 @@ describe(`GET:/api/users --sad path`, () => {
...
+describe(`GET /api/articles --sad path`, () => {
+  test("404: invalid path", () => {
+    return request(app)
+      .get("/invalid-path")
+      .expect(404)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Invalid path");
+      });
+  });
+});
```
Comment (douglashellowell) [inline app/__tests__/app.test.js]: no need to repeat this test!

you have this same test above

**Comment by douglashellowell** [comment]
You still need to use `toBeSortedBy()` in your `"200: responds with an an array of articles objects having eight properties"` test

merge once that's working!

**Review by Angaca — APPROVED** [review]
Good to go

**Pull request merged**